common/init.sls

```diff
@@ -45,8 +45,15 @@ host-{{ hostname }}:
 {% endfor %}
 
 {% for ssh_user in common.ssh_users %}
+{{ ssh_user }}:
+  user.present:
+    - home: /home/{{ ssh_user }}
+    - optional_groups:
+        - wheel
```

Comment:
AFAICT, OSX's BSD roots show through here and `wheel` does the usual thing. https://discussions.apple.com/thread/2210858?start=0&tstart=0